src/ui/windows/TogglDesktop/TogglDesktop/Toggl.cs

```diff
@@ -930,6 +929,7 @@ public static TogglRgbColor GetAdaptiveRgbColorFromHex(string hexColor, TogglAda
     public static readonly BehaviorSubject<DateTime> TimelineSelectedDate = new BehaviorSubject<DateTime>(DateTime.Today);
 
     public static readonly BehaviorSubject<TogglTimeEntryView?> RunningTimeEntry = new BehaviorSubject<TogglTimeEntryView?>(null);
+    public static IObservable<Unit> StoppedTimerState = RunningTimeEntry.Where(te => te == null).Select(_ => new Unit());

```

Comment:
minor: by convention `Unit.Default` is used instead of `new Unit()` (just to avoid using `new` keyword as much as possible, I suppose). Using `default` is also fine.